ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-ref.yaml

```diff
@@ -24,6 +27,9 @@ ref:
   - name: TEST_MODE
     documentation: Optional. The test mode of the workflow
     default: ""
+  - name: EVAL_CONFIG_ENVVARS
```

Comment:
Should this also start with `OO` like the other ENV vars? Is this one special?

Reply:
I didn't fully understand the logic behind `OO_` prefix but to me it looked like `OO_` prefixed variables control the OLM CRs (e.g. `OperatorGroup`, `Subscription`). Not `OO_` prefixed vars seemed to be configuration options for the step itself like `TEST_MODE` defined above.

Reply:
Ah okay that seems to make sense.